build.sbt

```diff
@@ -152,7 +152,7 @@ Compile / doc / scalacOptions ++= Seq(
   "ai.eto.rikai.sql.spark.parser"
 )
 
-assembly / assemblyJarName := s"${name.value}-assembly-${sparkVerStr.value}_${scalaBinaryVersion.value}-${version.value}.jar"
+assembly / assemblyJarName := s"${name.value}_${scalaBinaryVersion.value}-assembly-${sparkVerStr.value}-${version.value}.jar"
```

Comment:
I think keeping the `scalaBinaryVersion` after `name` and `sparkVerStr` is more idiomatic. It's like telling the audience `scalaBinaryVersion` is for both `Rikai` and `Spark`